product-service/functions/createProduct.ts

```diff
@@ -0,0 +1,50 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { productRepository } from '../repository';
+import {Product} from "../types";
+
+const baseResponse = {
+    headers: {
+        'Access-Control-Allow-Origin': '*',
+    }
+}
+
+function validate({title, description, price, author, count}: Omit<Product, 'id'>) {
+    // simple validation
+    return Number.isInteger(count) && count >=0 && Number.isInteger(price) && price >=0 && typeof title === 'string' && typeof description === 'string' && typeof author === 'string';
+}
+
+export const createProduct: APIGatewayProxyHandler = async (event, _context) => {
+    console.log(`CALL createProduct with body parameters: `, event.body);
```

Comment:
Please, add console.log to context and event params

Reply:
I hope console.log is temporary solution, anyway, we take parameters from event.body, not from query or context. I think that's enough for now

Reply:
You can see your log data in lambda console and cloudWatch logs. See logs info https://docs.aws.amazon.com/lambda/latest/dg/python-logging.html. But if it is enough for you log just event.body it can be so